src/bentoml_cli/deployment.py

```diff
@@ -27,9 +27,7 @@
 def raise_deployment_config_error(err: BentoMLException, action: str) -> t.NoReturn:
     if err.error_code == HTTPStatus.UNAUTHORIZED:
         raise BentoMLException(
-            f"{err}\n* BentoCloud sign up: https://cloud.bentoml.com/\n"
-            "* Login with your API token: "
-            "https://docs.bentoml.com/en/latest/bentocloud/how-tos/manage-access-token.html"
+            f"{err}\n* BentoCloud API token is required for authorization. Run `bentoml cloud login` command to sign up or create a token"
```

Comment:
It is a bit confusing. Why would `bentoml cloud login` sign up or create token? It is for logging in, right?

Reply:
How about change it to 
` Run `bentoml cloud login` command to login first`